models/iaf_wang_2002.h

```diff
@@ -0,0 +1,518 @@
+    k_1 &= -\alpha \tau_\mathrm{r} \mathrm{E_N} \Big[ \frac{\tau_\mathrm{r}}{\tau_\mathrm{d}}, \alpha \tau_\mathrm{r}
+\Big] + (\alpha \tau_\mathrm{r})^{\frac{\tau_\mathrm{r}}{\tau_\mathrm{d}}} \Gamma \Big[ 1 -
+\frac{\tau_\mathrm{r}}{\tau_\mathrm{d}} \Big]
+
+where :math:`\mathrm{E_N}` is the `generalized exponential integral
+<https://en.wikipedia.org/wiki/Exponential_integral#Generalization>`_, and :math:`\Gamma` is the `gamma function
+<https://en.wikipedia.org/wiki/Gamma_function>`_. For these values of :math:`k_0` and :math:`k_1`, the approximate model
+will approach the exact model for large t.
+
+The specification of this model differs slightly from the one in [1]_. The parameters :math:`g_\mathrm{AMPA}`,
+:math:`g_\mathrm{GABA}`, and :math:`g_\mathrm{NMDA}` have been absorbed into the respective synaptic weights.
+Additionally, the synapses from the external population is not separated from the recurrent AMPA-synapses.
+
+For more implementation details and a comparison to the exact version see:
+
+- `wong_approximate_implementation <../model_details/wong_approximate_implementation.ipynb>`_
+
+
+Parameters
+++++++++++
+
+The following parameters can be set in the status dictionary.
+
+
+=============== ======= ===========================================================
+ E_L            mV      Resting potential
+ E_ex           mV      Excitatory reversal potential
+ E_in           mV      Inhibitory reversal potential
+ V_th           mV      Threshold potential
+ V_reset        mV      Reset potential
+ C_m            pF      Membrane capacitance
+ g_L            nS      Leak conductance
+ t_ref          ms      Refractory period
+ tau_AMPA       ms      Synaptic time constant for AMPA synapse
+ tau_GABA       ms      Synaptic time constant for GABA synapse
+ tau_rise_NMDA  ms      Synaptic rise time constant for NMDA synapse
+ tau_decay_NMDA ms      Synaptic decay time constant for NMDA synapse
+ alpha          1/ms    Scaling factor for NMDA synapse
+ conc_Mg2       mM      Extracellular magnesium concentration
+ gsl_error_tol  -       GSL error tolerance
+=============== ======= ===========================================================
+
+
+Recordables
++++++++++++
+
+The following values can be recorded.
+
+=========== ===========================================================
+ V_m         Membrane potential
+ s_AMPA      sum of AMPA gating variables
+ s_GABA      sum of GABA gating variables
+ I_NMDA      NMDA current
+=========== ===========================================================
+
+.. note::
+   :math:`g_{\mathrm{\{\{rec,AMPA\}, \{ext,AMPA\}, GABA, NMBA}\}}` from [1]_ is built into the weights in this NEST
+model, so setting the variables is thus done by changing the weights.
+
+.. note::
+    For the NMDA dynamics to work, the both pre-synaptic and post-synaptic neuron must be of type iaf_wang_2002. For
+AMPA/GABA synapses, any pre-synaptic neuron can be used.
```

Comment:
Could you also add a note like

For technical reasons, spikes from `iaf_wang_2002` neurons must be recorded with `time_in_steps: True` set in the spike recorder, ignoring the offset value. We hope to correct this in a future version of NEST.